challenge-02/README.md

```diff
@@ -4,29 +4,39 @@ Nesse exercício, você está livre para escolher os nomes para suas variáveis
 
 ```js
 // Crie uma função que receba dois argumentos e retorne a soma dos mesmos.
-?
+function soma( x,y ){
+	return  x + y;
+}
+
 
 // Declare uma variável que receba a invocação da função criada acima, passando dois números quaisquer por argumento, e somando `5` ao resultado retornado da função.
-?
+
+var total =soma(15, 8) + 5;
 
 // Qual o valor atualizado dessa variável?
-?
+28
 
 // Declare uma nova variável, sem valor.
-?
+var semvalor;
 
 /*
 Crie uma função que adicione um valor à variável criada acima, e retorne a string:
     O valor da variável agora é VALOR.
 Onde VALOR é o novo valor da variável.
 */
-?
+function retornavalor(){
+  return retornavalor;
```

Comment:
@simonemyrrha, vou te acompanhar, ok? Olha só:

> Crie uma função que adicione um valor à variável criada acima, e retorne a string: "O valor da variável agora é VALOR"

Antes dessa função, você declarou uma variável chamada `semvalor`, correto? Agora você precisa fazer com que essa próxima função que você irá criar, adicione um valor à variável já criada acima. O valor será adicionado à variável assim que a função for invocada. Lembra como fizemos no video? Vou comentar as linhas pra ficar mais fácil de entender:

```js
// criando a variável:
var semvalor;

// criando a função
function retornavalor() {
  // aqui eu faço a variável criada acima receber um valor
  semvalor = 10;
  // e aqui, eu retorno a mensagem do enunciado, só trocando `VALOR` pela variável:
  return 'O valor da variável agora é ' + semvalor;
}
```

Ficou mais claro? Qualquer coisa me avise ;)